translations/ui-consortia-settings/en.json

```diff
@@ -39,6 +39,10 @@
   "consortiumManager.members.permissionSets.compare.permissionSet": "Permission set",
   "consortiumManager.members.permissionSets.compare.permissionSet.placeholder": "Select Permission set",
   "consortiumManager.members.permissionSets.compare.permissionSet.empty": "No items to show",
+  "consortiumManager.members.permissionSets.create.permissionSet.error": "Users were not successfully assigned to permission set. {error}",
+  "consortiumManager.members.permissionSets.create.permissionSet.success": "Users successfully assigned to permission set.",
```

Comment:
it's not correct message, you don't assign anything, you create permissions set

Reply:
This error messages from Jira ticket. I can improve it then

Reply:
my comment is not connected to error message, it's for success

Reply:
btw error message is connected to assign and not to permissions set save, you mixed up requirements for different actions